src/components/Main.jsx

```diff
@@ -206,6 +207,11 @@ function Main(props) {
 	const [showModal, setShowModal] = useState("close");
 
 	useEffect(() => {
+		const handleScroll = () => {
```

Comment:
Add feature for multiple browser using database to fetch token and authenticate user